src/client/firebase/auth.js

```diff
@@ -42,8 +42,7 @@ export async function signIn({ email, password }) {
 
 export async function signUp({ email, password }) {
   try {
-    await auth.createUserWithEmailAndPassword(email, password);
-    return true;
+    return await auth.createUserWithEmailAndPassword(email, password);
```

Comment:
You could also make it more clear what we are returning from here (nothing wrong with this code)

Just a suggestion:
```javascript
const user = await auth.createUserWithEmailAndPassword(email, password);
return user;
```
Then it is just easier to understand what gets returned 👍